public/locales/en/translation.json

```diff
@@ -91,7 +91,7 @@
       },
       {
         "header": "QUESTIONS AND COMMENTS",
-        "content": "Please feel free to send your privacy-related comments or questions to Privacy@MITRE.org. We will do our best to respond to all reasonable inquiries in a timely manner. For all other VCI™ related questions and comments, please contact us at vaccination@mitre.org."
+        "content": "Please feel free to send your privacy-related comments or questions to Privacy@MITRE.org. We will do our best to respond to all reasonable inquiries in a timely manner. For all other application related questions and comments, please contact us at vci-developers@mitre.org."
```

Comment:
Is there a reason the Privacy email address has a capitalized domain name, while the vci-developers has a lower-case one?

Reply:
it's copied from the public VCI.org site, don't know why they capitalized Privacy.

Reply:
Got it! Didn't realize it came directly from there.